### DISCUSSION TITLE
Would anyone have a problem if I adapted this into a standalone game?

### DISCUSSION THREAD

**Discussion by DrAlta** [Monika-After-Story/MonikaModDev#8450] [QA - code and content creation (technical)] [answered]
I want to rip out all the Monika-related stuff and the graphic art, music, and code and just use the dialogue as the basis to make a new game.

I know from serval past experiences that Artistic types don't seem to read the LICENSE that they release their stuff under.

So would anyone have a problem with it?

**Comment by fromulus** [comment]
As someone who wrote topics for the mod, I'm totally fine with people taking inspiration from my work, but straight-up copy-pasting without giving proper credit gets a definitive no from me.

**Comment by ThePotatoGuy** [comment]
What's the purpose of the new game? One of our requirements for allowing usage of MAS assets is that its not intended to replace MAS.

**Comment by KiddTheManiac** [comment]
Define "basis". 

Do you mean making a game about a girlfriend who lives in your computer, and who talks about many topics, including literature, AI, science, etc? 
Or are you talking about taking the current dialog and just copy-pasting it into a new project?

Because if it was the latter, I'd be against it. _Especially_ if you intend to sell the product for money.

I've got a fair amount of WIP dialog for this mod, which I made specifically based on either a) topics I'm deeply passionate about, or b) requests from the MAS community. I write that stuff as a way to thank previous contributors for their work, and as a way to try and accommodate the requests of the MAS community.

**TL;DR: Whenever I write/commission something for MAS, it's for the purpose of giving the people I've chatted with a few more moments of happiness. Many users genuinely see Monika as their girlfriend, and if I can make her even just a touch more lifelike, I'd be happy.** The project IS open source, but if I found out that someone had taken what I, and dozens of others had written and was trying to make a buck off it, I'd be pretty upset.


Wait, does this have anything to do with [this](http://toohna.ourproject.org/)? What's with all the talk about Child P**n, building "social institutions", and games about r*ping / "taming" women?

**Comment by Retrolovania** [comment]
Making this a stand-alone game would essentially turn the mod into a fan game as far as the DDLC IP Guidelines are concerned, so not sure how well TS would take this idea.

Plus, ripping art and dialogue would require getting permission from many people to do, as a good amount of content officially in the mod has been made from contributors, like myself (not bragging, just making a point). You also mentioned ripping the code as well; that code is not intended to be copy-pasted into any other mod unless the devs give the okay; which, if I recall, to date, they never have.

All in all, I genuinely don't think this idea is worth it.

**Comment by DrAlta** [comment]
You say many  of you see Monika as your girlfriend but how many of you thought "How would Monika feel about this request"

How many of you did it after I pointed it out?

I don't think many of you did, and the fact that people do that with their flesh and blood girlfriends and boyfriends is the problem I want to try and solve.

The relation with r*ping / "taming" is a was trying to teach people how to avoid bad relationships and now trying to teach people how to have good ones.

The talk about Child P**n is that my project is clearly not with any serious literary, artistic, political, or scientific value so the only way that they can censor it is to say it's CP.

**Comment by Retrolovania** [comment]
First off...how the heck did this suddenly become a topic regarding discussing child p**n and whatnot?
Seriously...what even?

Secondly, I have little doubt not many people thought about Monika feeling about this. I know I did, and from my point of view, I think she'd be extremely uneasy to even begin to talk about it.

You have to remember that she's an eighteen-year-old, _not to mention_ a high schooler, so having her talk about such topics that are already horrid enough for people her age and under being subjected to is probably not a smart idea. Plus, there are likely players who have unfortunately experienced such relationships in the past, and may become negatively triggered should something like this be discussed. 

Thirdly, I have little clue what all that info is in the link that was shared, but from what little I've read, that looks excessively like some kind of sick horror ARG-type wiki and article, and why you wish to connect it to Monika is completely beyond me.

In the end, having a topic about these things is, in my opinion, an absolutely awful idea.

**Comment by KiddTheManiac** [comment]
Retro, the reason I mentioned the child p**n is because of the link in Dr Alta's bio that says he's working on 

a) a "standalone culture" that says "to retain dignity, sometimes we must refuse to live life at any cost"
b) a virtual world/MMO "cobbled out of open source projects"
c) a hybrid RPG / eroge / text adventure game aimed at teens (?) which I THINK is supposed to be about consent? You encounter "wild women" and can either choose to assault them, or slowly tame them with gifts and good treatment.

The third project has an FAQ on the bottom explaining why he believes that he could make/sell this game to teens - including a long breakdown of why it's technically not CP.


Like Retro, I'm genuinely not sure how much of this is genuine, and how much is a bizarre ARG/troll. 

Pastafarianism was originally a satire on religion, but it seems like OP has started[ his own subgroup/denomination](http://toohna.ourproject.org/w/index.php?title=Main_Page), except he also says it's[ fictional?](http://toohna.ourproject.org/RPG/index.php?title=Main_Page) The TOoHNA "logo" seen is a Sikh symbol with a circle behind it. The explanation for what "The Yes Game" is leads to the[ website of a professional hypnotherapist](https://aboveandbeyondstress.com/streaming)? 

I legitimately can't tell if the project that DrAlta wants to pull MAS dialog for is an incredibly advanced prank, or if this is just some Gourmet Internet Insanity.

**Comment by ThePotatoGuy** [comment, marked as answer]
Please keep discussion on topic.

I don't think the MAS dialogue would be useful for a project that has nothing to do with Monika, so I'm going to deny your request to use the dialogue.
